Gemfile

```diff
@@ -35,7 +35,7 @@ group :development, :test do
   gem "sprockets"
   gem "state_machines"
   gem "activerecord-typedstore"
-  gem "sqlite3", "~>1.4"
+  gem "sqlite3"
```

Comment:
I'm hoping the issue in https://github.com/Shopify/tapioca/pull/1875/ (which I don't know) is resolved by now

Reply:
Yes, this has been resolved in Rails.